arch/arm/boot/dts/overlays/imx290_327-overlay.dtsi

```diff
@@ -16,28 +16,7 @@
 			#size-cells = <0>;
```

Comment:
I can see that they aren't new files, but I'm not that keen on `*-overlay.dtsi` as filenames. And the relationship between this and the new `imx290_327.dtsi` isn't obvious, especially since neither `imx290_327.dtsi` nor `imx477_378.dtsi` appears to be used.

Reply:
Sorry, `imx290_327.dtsi` is used, and `imx477_378.dtsi` isn't even a thing, but the rest of the comment stands.